### PR TITLE
Fix private chat messages

### PR DESCRIPTION
**Pull request by jfsiebel**
fix the problem with private chat messages, improves the handling of HTML5 direct messages from #5342

### CODE REVIEW

File: bigbluebutton-html5/imports/api/common/server/helpers.js [hunk 1]
```diff
@@ -17,7 +18,13 @@ export const processForHTML5ServerOnly = fn => (message, ...args) => {
...
   const { routing } = envelope;
 
-  const shouldSkip = routing.msgType === MSG_DIRECT_TYPE && routing.userId !== NODE_USER;
+  const selector = {
+    userId: routing.userId,
+  };
+
+  const user = Users.findOne(selector);
+
+  const shouldSkip = routing.msgType === MSG_DIRECT_TYPE && (routing.userId !== NODE_USER && user.clientType !== 'HTML5');
```
Comment (antobinary) [inline bigbluebutton-html5/imports/api/common/server/helpers.js]: Please also check that we have such user in db' `const shouldSkip = user && routing.msgType === MSG_DIRECT_TYPE && (routing.userId !== NODE_USER && user.clientType !== 'HTML5');`

File: bigbluebutton-html5/imports/api/common/server/helpers.js [hunk 2]
```diff
@@ -17,7 +18,13 @@ export const processForHTML5ServerOnly = fn => (message, ...args) => {
   const { envelope } = message;
   const { routing } = envelope;
 
-  const shouldSkip = routing.msgType === MSG_DIRECT_TYPE && routing.userId !== NODE_USER;
+  const selector = {
+    userId: routing.userId,
```
Comment (antobinary) [inline bigbluebutton-html5/imports/api/common/server/helpers.js]: Please add meetingId: meetingId from envelope to narrow down search

File: bigbluebutton-html5/imports/api/common/server/helpers.js (hunk 1 above)
Comment (oswaldoacauan) [reply bigbluebutton-html5/imports/api/common/server/helpers.js]: Hmm, are the collection always ready in this step? Like authentication/guest stuff

Comment (antobinary) [reply bigbluebutton-html5/imports/api/common/server/helpers.js]: They should be. This is more on a mid-meeting event, not an initializing one

**Pull request merged**